bot/seasons/evergreen/recommend_game.py

```diff
@@ -0,0 +1,54 @@
+import json
+import logging
+from pathlib import Path
+from random import shuffle
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+DIR = Path("bot/resources/evergreen/game_recs")
+game_recs = []
+
+# Populate the list `game_recs` with resource files
+for file_url in DIR.glob("*.json"):
+    with Path(file_url).open(encoding='utf-8') as file:
```

Comment:
`Path.glob()` will yield `Path` objects.

```suggestion
    with rec_path.open(encoding='utf-8') as file:
```